xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -227,7 +227,7 @@ void CGUIWindowVideoBase::OnInfo(CFileItem* pItem, ADDON::ScraperPtr& scraper)
   // so take a copy of the item now
   CFileItem item(*pItem);
   bool fromDB = false;
-  if (item.IsVideoDb() && item.HasVideoInfoTag())
+  if (item.IsVideoDb() || (item.HasVideoInfoTag() && item.GetVideoInfoTag()->m_iDbId != -1))
```

Comment:
Are you sure that the conditions (OR) are correct here, because we need a valid video info tag within the statement block.